ddtrace/bootstrap/sitecustomize.py

```diff
@@ -2,40 +2,30 @@
 Bootstrapping code that is run when using the `ddtrace-run` Python entrypoint
 Add all monkey-patching that needs to run by default here
 """
-
+import logging
 import os
 import imp
 import sys
-import logging
 
 from ddtrace.utils.formats import asbool, get_env, parse_tags_str
 from ddtrace.internal.logger import get_logger
 from ddtrace import config, constants
+from ddtrace.tracer import debug_mode, DD_LOG_FORMAT
 
-DD_LOG_FORMAT = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s".format(
-    "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s"
-    " dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] "
-    if config.logs_injection
-    else ""
-)
 
 if config.logs_injection:
     # immediately patch logging if trace id injected
     from ddtrace import patch
 
     patch(logging=True)
 
-debug = os.environ.get("DATADOG_TRACE_DEBUG")
-
-# Set here a default logging format for basicConfig
 
 # DEV: Once basicConfig is called here, future calls to it cannot be used to
 # change the formatter since it applies the formatter to the root handler only
 # upon initializing it the first time.
 # See https://github.com/python/cpython/blob/112e4afd582515fcdcc0cde5012a4866e5cfda12/Lib/logging/__init__.py#L1550
-if debug and debug.lower() == "true":
-    logging.basicConfig(level=logging.DEBUG, format=DD_LOG_FORMAT)
-else:
+# Debug mode from the tracer will do a basicConfig so only need to do this otherwise
+if not debug_mode:
     logging.basicConfig(format=DD_LOG_FORMAT)
```

Comment:
this will probably fail if they don't have `config.logs_injection` enabled, since the format is fixed.

Reply:
Ah yes, we still have to do the check for logs injection here.